xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/XWiki/SolrSearchAdminIndexingUserClass.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.1" encoding="UTF-8"?>
```

Comment:
I feel it would make more sense to create this class through a mandatory class initializer since you have several Java codes expecting this class to exist, including a mandatory document initializer.

Reply:
This class is now created by its own initializer.